python/graphscope/client/rpc.py

```diff
@@ -96,6 +96,7 @@ def __init__(self, endpoint):
     def waiting_service_ready(self, timeout_seconds=60, enable_k8s=True):
         begin_time = time.time()
         request = message_pb2.HeartBeatRequest()
+        response = None
```

Comment:
No need for that.

Reply:
This is for line-107 in finally statement: `if response is not None`.

When Ctrl-C signal occurs, The try statement cannot guarantee that `response` is always defined.

Reply:
Thanks for clearfication. So leave a comment hear please to prevent this line being drop by accident: `KeyboardInterrupt` is not an `Exception`, rather, it inherits from `BaseException`.